docs/CONTRIBUTING.md

```diff
@@ -7,29 +7,88 @@ sure you follow the steps below:
 - Use [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/)
   with `#<your issue number>` as the context for all commits
 - Make sure you are not adding any merge commits to your branch
-- Open a pull request & wait for a review!🩵
+- Open a pull request, include the issue it relates to in the body, for example:
+  `Closes #69`
+- Wait for a review! 🩵
 
-## Developing environment
+For more information on how to get started, read on!
 
-After installing all dependencies with `yarn`, you can run
+## Setting Up Your Build Environment
 
-```sh
-yarn dev
-```
+First install the required build dependencies using your system package manager
 
-to run the development server for Vivify on port 3000 that will (1)
-automatically reload when you make changes to the code, and (2), unlike the
-installed version, not shut down when there are no connected clients.
+- yarn
+- make
+- zip
 
-With the development server running, use
+Next you need to make sure you have Node.js installed
 
-```sh
-yarn viv
-```
+Node.js may be available with your OS or in your distro package manager.
+Alternatively you can install the latest version of Node.js using **[Node
+Version Manager](https://github.com/nvm-sh/nvm)** (**nvm**)
 
-instead of your installed Vivify executable. This will (1) connect to the
-development server on port 3000 instead of running the installed server, and (2)
-use the `viv` executable in the repository.
+> [!NOTE]
+> The version of **Node.js** shipped with some Linux distributions will fail to
+> build **Vivify**, in that case you should refer to the **nvm** documentation
+> to install the latest version
+
+## Running Vivify for Development
+
+First clone and open the **Vivify** repository
+
+Then run `yarn` to download Node.js dependencies
```

Comment:
Let's also combine these into 1 line now that there's nothing left in between :) Can all happen in that squashed commit so that the total commits here is 2 ^^